build-one.bash

```diff
@@ -4,6 +4,8 @@ set -ex
 [[ $# -eq 1 ]] || { echo "Usage: $0 KERNEL_NAME" >&2; exit 1; }
 BASE="$(readlink -f "$(dirname "$(readlink -f "$0")")")"
 KERNEL_DIR="$BASE/kernels/$1"
+# For use in kernel-specific do.bash files
+export KERNEL_DIR
```

Comment:
In the past, this has just been added to the individual do.bash file:

https://github.com/WireGuard/android-wireguard-module-builder/commit/8391037f86b95a4986b23b61f6f32efa760dd8a6

https://github.com/WireGuard/android-wireguard-module-builder/commit/4e217ddb576884a70fa3cdc08dc96da02dcddb37

I do not have a strong opinion though.

Reply:
Thanks! Indeed I didn't look into the history.